### ISSUE TITLE
Supplying * asterix into tag selection UI fails with blank screen

### ISSUE THREAD

**Issue by mad-wnn** [sulu/sulu#5441]
| Q | A
| --- | ---
| Sulu Version | 2.1.0
| Browser Version | Firefox 68.2

# Actual Behavior
supplying `*` into `tag_selection` UI fails with blank screen

browser console output:
```
SyntaxError: "nothing to repeat"
    <anonymous> Suggestion.js:28
    value AutoCompletePopover.js:116
    value AutoCompletePopover.js:114
    value Suggestion.js:82
```

**Comment by alexander-schranz** [comment]
@mad-wnn thank you for report. This seems to be also an issue in the 2.0 version. The `*` seems not be escaped for reges at https://github.com/sulu/sulu/blob/bfc18fd0a2a37db25c16ac9d21c76ee984ae7cec/src/Sulu/Bundle/AdminBundle/Resources/js/components/AutoCompletePopover/Suggestion.js#L28. And that why it fails.

I think we should use [replaceAll](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) instead of a regular expression here, would require to core-js polyfill as it seems Edge does not yet support it. /cc @danrot what do you think? A package for escape string for regex seems to be an overhead for me and replaceAll polyfill would be something we could remove in future. 

Do you want to create a pull request on the `release/2.0` branch to fix this?

**Comment by danrot** [comment]
Sorry, but I cannot even reproduce that error. I have tried to enter stuff like "asdf*" or only "*" into the tags field in the page excerpt tab, and at least there it seems to work for me. Am I missing something? Tried it in Chrome and Firefox.

**Comment by alexander-schranz** [comment]
@danrot You need first create a Tag with has a asterix inside it over `Settings` -> `Tags` and then go back on a page and search for `*`. The suggestion.js will try to highlight the `*` in that autocomplete dropdown and will crash.

**Comment by danrot** [comment]
Okay, got it, I get the following error now, that makes totally sense:

```
SyntaxError: Invalid regular expression: /*/: Nothing to repeat
```

What @alexander-schranz suggested sounds good to me, I don't see why we shouldn't be able to use `replaceAll`.

**Comment by mad-wnn** [comment]
Hello.May look into it in 3 days for more insight as I haven't got any tags containing asterixes, just ASCII strings.

**Comment by alexander-schranz** [comment]
This issue seems still exist and seems not be fixed by #6191.

Reproducer:

 - Create a Tag with `Tag *`
 - Open Tag Selection type as first value `*`
 - Admin UI crash
 
 Tested Sulu Version 2.3.4 (https://sulu.rocks/admin)

**Comment by alexander-schranz** [comment]
Fixed by #6239
